app/frontend/src/components/admin/AdminPage.vue

```diff
@@ -27,8 +28,18 @@ export default {
   },
   data() {
     return {
+      index: 2,
```

Comment:
Is `index` needed here?

Reply:
I forgot to remove it. I was trying to work on the tab issue

Reply:
Oh, OK - maybe I should wait until you're done!